Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc

```diff
@@ -131,7 +130,7 @@ void DDHGCalEEAlgo::execute(DDCompactView& cpv) {
   copies_.clear();
```

Comment:
copies_ is used only for debugging. Please, move it.

Reply:
Taken care of this